src/components/MultiButtonWithSubQuery.tsx

```diff
@@ -15,7 +15,21 @@ export const navigateWithSubQuery = ({
 }: {
   navigation: StackNavigationProp<any, string>;
   title?: string;
-  params?: { routeName: string; webUrl: string } | string;
+  params?:
+    | {
+        routeName: string;
+        webUrl: string;
```

Comment:
`webUrl` should be changed to an optional param here, right?

Reply:
I left this part of the code as it was before because I didn't want to have incompatibility with older versions later. If there is no webUrl, we don't show the first button anyway. When you say optional, do you mean it can be `undefined`? Then I think I need to change the following code as well.

```
 {!!subQuery && !!subQuery.routeName && (!!subQuery.webUrl || subQuery.params) && (
        <Button
          title={subQuery.buttonTitle || `${title} öffnen`}
          onPress={() => navigateWithSubQuery({ navigation, subQuery, rootRouteName, title })}
        />
      )}
```

Reply:
ok, keep it as is